src/views/Gallery.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+    <div id="gallery-content">
+        <div id="header-mobile">
+            <div>
+                <h2>Specimen gallery</h2>
+                <h3>Self similarity telescope</h3>
+            </div>
+            <div type="button" id="change-button">
+                <span
+                    alt="Swap icon"
+                    id="change-icon"
+                    class="material-icons-sharp"
+                    >swap_horiz</span
+                >
+                <p id="change-text">Change Visualizer</p>
+            </div>
+        </div>
+
+        <h1 id="header">Specimen gallery</h1>
+
+        <div type="button" class="visualizer-bar">
+            <h2>Self similarity telescope</h2>
+            <span class="material-icons-sharp">keyboard_arrow_up</span>
+        </div>
+        <div class="gallery">
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+        </div>
+
+        <div type="button" class="visualizer-bar">
+            <h2>Self similarity telescope</h2>
+            <span class="material-icons-sharp">keyboard_arrow_up</span>
+        </div>
+        <div class="gallery">
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+        </div>
+    </div>
+</template>
+
+<script setup lang="ts">
+    import SpecimenCard from '../components/SpecimenCard.vue'
+
+    const sequenceName = '2-acid val of Z'
+    const cardName = 'Pinstripe'
+</script>
+
+<style lang="scss" scoped>
+    p,
```

Comment:
Is anything from scss actually being used in this css block? I didn't see anything that didn't look like straight CSS. If there isn't, please remove the scss designation, as it's a little misleading and we want to keep things as simple as is practical. Thanks.

Reply:
Change has been implemented.